cmd/common/display/message_test.go

```diff
@@ -169,6 +169,43 @@ func Example_respTxQuery_json() {
 	// }
 }
 
+func Example_empty_respTxQuery_text() {
+	txResp := getExampleTxQueryResponse()
+	txResp.Tx = transactions.Transaction{}
+	Print(&RespTxQuery{Msg: txResp}, nil, "text")
+	// Output:
+	// Transaction ID: 31303234
+	// Status: success
+	// Height: 10
+	// Log: This is log
+}
+
+func Example_empty_respTxQuery_json() {
+	txResp := getExampleTxQueryResponse()
+	txResp.Tx = transactions.Transaction{}
+	Print(&RespTxQuery{Msg: txResp}, nil, "json")
+	// Output:
+	// {
+	//   "result": {
+	//     "hash": "31303234",
+	//     "height": 10,
+	//     "tx": {
+	//       "Signature": null,
+	//       "Body": null,
+	//       "Serialization": "",
+	//       "Sender": null
+	//     },
```

Comment:
This is what should happen in `kwil-cli` for one of these unconfirmed tx responses. @charithabandi Is there a quick test you can do?

Reply:
probably run it with the stress tool ? with high concurrency?